changelog/_unreleased/2020-09-17-improve-line-item-custom-field-rule-execution.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Improve line item custom field rule execution
+issue: NEXT-10858
+author: Uwe Kleinmann
+author_email: u.kleinmann@kellerkinder.de
+author_github: @kleinmann
+---
+# Core
+* Fix line item custom field rule condition validation for boolean custom fields to allow for unchecked values.
```

Comment:
changelog entries are not valid according to the conventions 😉

Reply:
Can you be a bit more specific? Are file paths a required part of changelog entries? I'm not sure what's actually missing.
I removed the sections that are not applicable to the change. Upgrade information is also not needed, as data-wise there are no changes.

Reply:
Never mind, I think I got it 🙂 Changed the descriptions accordingly and they're now using "Added" as the action :-)

Reply:
have a look here 😉 https://github.com/shopwareBoostDay/platform/tree/master/changelog